src/main/java/mate/academy/dto/CreateBookRequestDto.java

```diff
@@ -2,13 +2,25 @@
 
 import java.math.BigDecimal;
 import lombok.Data;
+import mate.academy.validator.Author;
+import mate.academy.validator.CoverImage;
+import mate.academy.validator.Description;
+import mate.academy.validator.Isbn;
+import mate.academy.validator.Price;
+import mate.academy.validator.Title;
 
 @Data
 public class CreateBookRequestDto {
+    @Title
     private String title;
+    @Author
     private String author;
+    @Isbn
```

Comment:
It's good to have Isbn validator and URL validator, bu for other fields you can use `@NotBlank(message...) ` for String type and for BigDecimal price:
```
@NotNull(message = "Price must not be null")
@Positive(message = "Price must not be negative")
```